compute_endpoint/globus_compute_endpoint/endpoint/config/utils.py

```diff
@@ -82,6 +82,13 @@ def load_config_yaml(config_str: str) -> Config:
     config_dict = config_schema.dict(exclude_unset=True)
 
     try:
+        if config_dict.get("multi_user") is True:
+            # Temporary hack to not instantiate executor for MU; necessary due to the
+            # historical design to default-instantiate an executor in the Config
+            # object; this hack will no longer be necessary when MEP and UEP configs
+            # are cleanly separated rather than the current sharing of the same basal
+            # data structure
+            config_dict["executors"] = []
```

Comment:
I think this deserves its own PR.

Reply:
*nod; fair.  It came about because of the thought-process of the tests and their splatter, but I can make this it's own PR as well.

Reply:
Done: #1704

Once that lands, I'll rebase this PR upon `main` ... and then this comment becomes stale!  Thanks for the nudge.

Reply:
Okay, rebased.  Thanks for the quick review!